src/main/java/ch/njol/skript/expressions/ExprRemainingAir.java

```diff
@@ -52,8 +58,40 @@ protected String getPropertyName() {
 	}
 	
 	@Override
-	public Timespan convert(final LivingEntity e) {
-		return Timespan.fromTicks(e.getRemainingAir());
+	public Timespan convert(final LivingEntity entity) {
+		return Timespan.fromTicks_i(entity.getRemainingAir());
+	}
+	
+	@Nullable
+	@Override
+	public Class<?>[] acceptChange(Changer.ChangeMode mode) {
+		return CollectionUtils.array(Timespan.class);
+	}
+	
+	@SuppressWarnings("null")
+	@Override
+	public void change(Event event, @Nullable Object[] delta, Changer.ChangeMode mode) {
+		long ticks = ((Timespan)delta[0]).getTicks_i();
+		switch (mode) {
+			case ADD:
+				for (LivingEntity entity : getExpr().getArray(event))
+					entity.setRemainingAir(entity.getRemainingAir() + (int) ticks);
+				break;
+			case REMOVE:
+				for (LivingEntity entity : getExpr().getArray(event))
+					entity.setRemainingAir(entity.getRemainingAir() - (int) ticks);
+				break;
+			case SET:
+				for (LivingEntity entity : getExpr().getArray(event))
+					entity.setRemainingAir((int) ticks);
+				break;
+			case DELETE:
+			case REMOVE_ALL:
+			case RESET:
+				for (LivingEntity entity : getExpr().getArray(event))
+					entity.setRemainingAir(0);

```

Comment:
Shouldn't reset set it to 20?

Reply:
Umm yeah, it should do that. 0 could make a player suffocate without water for a tick or so.